src/main/java/model/io/JSONModelReader.java

```diff
@@ -0,0 +1,25 @@
+package model.io;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import model.structures.Model;
+
+import java.io.File;
+import java.io.IOException;
+
+public class JSONModelReader {
```

Comment:
Czy to jest potrzebne? Jak tak to jak tego się używa i do czego?

Reply:
teraz już nie jest